static/app/components/replays/replayController.tsx

```diff
@@ -81,6 +81,7 @@ function ReplayOptionsMenu({
           icon={<IconSettings size="sm" />}
         />
       )}
+      disabled={disableSettings}
```

Comment:
Not great that it opens anyway when disabled=true 

Maybe we can remove the whole component instead?

Reply:
that was an error on my part - this works as intended now (no pop up when disabled)